apps/invitations/services.py

```diff
@@ -60,7 +67,8 @@ def process_user_completed(self, user):
         """
         email = user.email
         if email:
-            q = self.model.objects.filter(email=email)
-            if q.exists():
-                invitation = q.first()
-                invitation.complete(user)
+            for model in self.models.values():
```

Comment:
```suggestion
            for model in self.models:
```

Just go with a simple list, see below.